tests/noaa/hrrr/forecast_48_hour/dynamical_dataset_test.py

```diff
@@ -12,6 +16,129 @@ def dataset() -> NoaaHrrrForecast48HourDataset:
     return NoaaHrrrForecast48HourDataset(primary_storage_config=NOOP_STORAGE_CONFIG)
 
 
+@pytest.mark.slow
+def test_backfill_local_and_operational_update(
+    dataset: NoaaHrrrForecast48HourDataset, monkeypatch: pytest.MonkeyPatch
+) -> None:
```

Comment:
HRRR has an Icechunk replica (correct me if I'm wrong if we don't plan to keep that); would it be worth including an icechunk replica in the integration test or is this too slow? (Maybe not worth it -- maybe we use keep the GFS one as our icechunk replica integration test)